components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -820,14 +814,6 @@ void AdsServiceImpl::OnShow(
   auto type = ToMojomNotificationEventType(ads::NotificationEventType::VIEWED);
 
   bat_ads_->OnNotificationEvent(id, type);
-
-  // If we've surpassed the maximum number of visible notifications,
-  // then close the oldest one
-  notifications_.push_back(id);
-  if (kMaximumAdNotifications > 0 &&
-      notifications_.size() > kMaximumAdNotifications) {
-    CloseNotification(notifications_.front());
-  }
```

Comment:
This wasn't needed, as the notification tracking was already occurring at a lower level in the implementation.